game/src/main/java/net/demilich/metastone/game/spells/desc/condition/StorageContainsCondition.java

```diff
@@ -0,0 +1,25 @@
+package net.demilich.metastone.game.spells.desc.condition;
+
+import net.demilich.metastone.game.GameContext;
+import net.demilich.metastone.game.Player;
+import net.demilich.metastone.game.cards.Card;
+import net.demilich.metastone.game.cards.CardList;
+import net.demilich.metastone.game.entities.Entity;
+import net.demilich.metastone.game.spells.custom.EnvironmentEntityList;
+
+public class StorageContainsCondition extends Condition {
```

Comment:
Source v. Target in these sorts of situations is confusing. Add a `javaDoc` here.